manifest.json

```diff
@@ -28,5 +28,20 @@
   },
   "permissions": [
     "tabs"
-  ]
+  ],
+  "applications": {
+    "gecko": {
+      "strict_min_version": "55"
```

Comment:
There is another small issue with `npm run lint`, complaining about `strict_min_version`:

```
"/applications/gecko/strict_min_version" should match pattern "^[0-9]{1,3}(\.[a-z0-9]+)+$"
```

Could you fix this as well please?